assets/airports/airportLoadList.js

```diff
@@ -257,6 +257,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             name: 'Vienna International Airport',
             wip: false
         },
+        {
+            icao: 'lrop',
+            level: 'medium',
+            name: 'Bucharest Henri Coanda International Airport',
+            wip: false
```

Comment:
Yessir! Only the most fully refined airports will have a `wip: false`. See #179 for more on this.